hugegraph-core/src/main/java/org/apache/hugegraph/masterelection/StandardRoleElectionStateMachine.java

```diff
@@ -49,7 +53,11 @@ public void apply(StateMachineCallback stateMachineCallback) {
         while (!this.shutdown) {
             E.checkArgumentNotNull(this.state, "State don't be null");
             try {
+                RoleState pre = this.state;
                 this.state = state.transform(context);
```

Comment:
Note: `state.transform` is not an atomic operation

Reply:
the machine run in single thread

Reply:
taskManager and globalMadterInfo are use lock to sync